### DISCUSSION TITLE
Profile 7 MEL vs Profile 8.1

### DISCUSSION THREAD

**Discussion by joaoserra** [quietvoid/dovi_tool#272] [General]
Hi, nor a question about dovi_tool, but a question about DV.

I though that profile 7 without any FEL and NLQ data was basically the same as profile 8.1 because since both have a PQ 10bit HDR10 BL , there's really no need for any MMR/polynomial reshapping, but when you convert 7 to 8.1 using your tool, you have a comment in your source that says "Converts the RPU to be profile 8.1 compatible.
    ///          Both luma and chroma mapping curves are set to no-op.
    ///          This mode handles source profiles 5, 7 and 8."

So, what is dolby trying to achieve here? That is the purpose of these luma/choma mappings in profile 7?

Thanks!

**Comment by quietvoid** [comment]
I'm not sure what the purpose of polynomial/MMR mapping is for profile 7.

**Comment by joaoserra** [comment]
Justo to confirm: These curves only present in FEL, not on MEL. 
So, a native MEL bluray would not have this curves, right?

**Reply by quietvoid** [reply]
Well technically they could but I don't think I've seen any.

**Comment by joaoserra** [comment]
Ok. Thanks.
So, based on what you have seen, is there any meaningful difference in metadata from a native profile 7 MEL to profile 8.1?

**Reply by quietvoid** [reply]
No.

**Reply by joaoserra** [reply]
Ok. Thanks for the info!
